### PR TITLE
Student House CRUD

### PR DESCRIPTION
**Pull request by Samarasan**
Updated fields in DB

### CODE REVIEW

File: server/database/admin/studentHouseinfo.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                }
+                logger.info(`End SQL Execution for create StudentHouse`);
+                resolve(results);
+            });
+        }),
+
+    getAllstudentHouse: () =>
+        new Promise((resolve: any, reject: any) => {
+            logger.info(`Begin SQL Execution for get Exam By school id `);
+              const query: any = `SELECT * FROM StudentHouse WHERE active = 1`;
```
Comment (ArunprasadRajendran) [inline server/database/admin/studentHouseinfo.ts:35]: include school_id in the where condition

File: server/database/admin/studentHouseinfo.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                }
+                logger.info(`End SQL Execution for get StudentHouse By school id `);
+                resolve(results);
+            });
+        }),
+
+    getAllstudentHouseById: (id: string) =>
+        new Promise((resolve: any, reject: any) => {
+            logger.info(`Begin SQL Execution for get StudentHouse By id `);
+            const query: any = `SELECT * FROM Admin where id = ${id}`;
```
Comment (ArunprasadRajendran) [inline server/database/admin/studentHouseinfo.ts:50]: table name is incorrect

File: server/database/admin/query.ts [hunk 3]
```diff
@@ -583,3 +584,26 @@ function updateNewlineCharacter(address: string) {
...
+      const query: any = `INSERT INTO StudentHouse( school_id,name,academic_year,color, created_time, created_by,active) VALUES
+         (${data.school_id},'${data.name}','${data.academic_year}','${data.color}','${data.created_time}',${data.created_by},1)`;
+        return query;
+    },
+    updateStudentHouse: (data: StudentHouse) => {
+        const query: any = `UPDATE StudentHouse
+        SET school_id = ${data.school_id},
+        name =  ${data.name},
+        color =  ${data.color},
+        academic_year =  ${data.academic_year},
```
Comment (ArunprasadRajendran) [inline server/database/admin/query.ts:605]: remove the comma here for last field else update query will not work

**Review by ArunprasadRajendran — CHANGES_REQUESTED** [review]

File: server/database/admin/query.ts (hunk 3 above)
Comment (Samarasan) [reply server/database/admin/query.ts]: Updated

File: server/database/admin/studentHouseinfo.ts (hunk 2 above)
Comment (Samarasan) [reply server/database/admin/studentHouseinfo.ts]: Updated

**Review by ArunprasadRajendran — APPROVED** [review]

**Pull request merged**